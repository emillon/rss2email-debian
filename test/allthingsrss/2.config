[DEFAULT]
to = a@b.com
date-header = True
post-process = rss2email.post_process.downcase downcase_message
