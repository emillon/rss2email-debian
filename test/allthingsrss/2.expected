SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Version 2.67 Released
Date: Tue, 21 Sep 2010 16:55:07 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=62
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/aGbf5Gefkmc/
X-RSS-TAGS: Uncategorized

version 2.67 of rss2email is now available for both [linux](http://www.allthingsrss.com/rss2email/rss2email-2.67.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.67.zip), which includes the latest development version of feedparser. changes from the previous version:

  * fixed entries that include an id which is blank (i.e., an empty string) were being resent
  * fixed some entries not being sent by email because they had bad from headers
  * fixed from headers with html entities encoded twice
  * compatibility changes to support most recent development versions of feedparser
  * compatibility changes to support google reader feeds

complete list in the official [changelog](http://www.allthingsrss.com/rss2email/changelog).

[![](http://feedads.g.doubleclick.net/~a/sjq8v-lq2dnkzkelsypic4fjplm/0/di)](http://feedads.g.doubleclick.net/~a/sjq8v-lq2dnkzkelsypic4fjplm/0/da)  
[![](http://feedads.g.doubleclick.net/~a/sjq8v-lq2dnkzkelsypic4fjplm/1/di)](http://feedads.g.doubleclick.net/~a/sjq8v-lq2dnkzkelsypic4fjplm/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/agbf5gefkmc)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/agbf5gefkmc/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Version 2.68 Released with Actual New Features
Date: Fri, 01 Oct 2010 18:21:26 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=72
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/bT-I0iH2vw8/
X-RSS-TAGS: Uncategorized

unlike the last few versions of rss2email that have trickled out, i finally got around to adding a few new oft-requested features! version 2.68 of rss2email is now available for both [linux](http://www.allthingsrss.com/rss2email/rss2email-2.68.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.68.zip).

changes from the previous version:

  * added ability to pause/resume checking of individual feeds through pause and unpause commands
  * added ability to import and export opml feed lists through importopml and exportopml commands

complete list in the official [changelog](http://www.allthingsrss.com/rss2email/changelog).

**pause/unpause**

through `r2e pause _n_` where _n_ is a feed number, you can temporarily suspend checking that feed for new content. to start checking it again, simply run `r2e unpause _n_`. when you `r2e list`, an asterisk indicates that the feed is currently unpaused and active.

**ompl import/export**

[opml](http://en.wikipedia.org/wiki/opml) is an xml format commonly used to exchange a list of rss feeds between applications. `r2e opmlexport > _filename_` will give you a file that you can use to import your list of feeds into another application. if you've exported feeds from another application into a file, `r2e opmlimport _filename_` will add those feeds to your rss2email feed list.

[![](http://feedads.g.doubleclick.net/~a/nygtsiuss9pmvrz6092xgghnnkg/0/di)](http://feedads.g.doubleclick.net/~a/nygtsiuss9pmvrz6092xgghnnkg/0/da)  
[![](http://feedads.g.doubleclick.net/~a/nygtsiuss9pmvrz6092xgghnnkg/1/di)](http://feedads.g.doubleclick.net/~a/nygtsiuss9pmvrz6092xgghnnkg/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/bt-i0ih2vw8)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/bt-i0ih2vw8/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: How to Read RSS Feeds in Emacs
Date: Fri, 05 Nov 2010 17:36:14 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=79
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/JryfOLe_q6c/
X-RSS-TAGS: Uncategorized

emacs and rss2email user erik hetzner has written up a tutorial on how he integrated [rss feed reading into emacs](http://www.emacswiki.org/emacs/howtoreadfeedsinemacsviaemail) using rss2email.

[![](http://feedads.g.doubleclick.net/~a/w185ejeikknpfrcwp9sv0gdrfjk/0/di)](http://feedads.g.doubleclick.net/~a/w185ejeikknpfrcwp9sv0gdrfjk/0/da)  
[![](http://feedads.g.doubleclick.net/~a/w185ejeikknpfrcwp9sv0gdrfjk/1/di)](http://feedads.g.doubleclick.net/~a/w185ejeikknpfrcwp9sv0gdrfjk/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/jryfole_q6c)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/jryfole_q6c/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Version 2.69 Released
Date: Fri, 12 Nov 2010 18:45:08 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=85
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/ubaE55Le1OU/
X-RSS-TAGS: Uncategorized

version 2.69 of rss2email is now available for both [linux](http://www.allthingsrss.com/rss2email/rss2email-2.69.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.69.zip).

changes from the previous version:

  * added support for connecting to smtp server via ssl, see smtp_ssl option 
  * improved backwards compatibility by fixing issue with listing feeds when run with older python versions 
  * added selective feed email overrides through override_email and default_email options 
  * added no_friendly_name to from from address only without the friendly name 
  * added x-rss-url header in each message with the link to the original item 

complete list in the official [changelog](http://www.allthingsrss.com/rss2email/changelog).

[![](http://feedads.g.doubleclick.net/~a/ygs5ivjpi9xnnjtza1sftazptua/0/di)](http://feedads.g.doubleclick.net/~a/ygs5ivjpi9xnnjtza1sftazptua/0/da)  
[![](http://feedads.g.doubleclick.net/~a/ygs5ivjpi9xnnjtza1sftazptua/1/di)](http://feedads.g.doubleclick.net/~a/ygs5ivjpi9xnnjtza1sftazptua/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/ubae55le1ou)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/ubae55le1ou/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Minor Correction to v2.69 Packages
Date: Fri, 10 Dec 2010 17:47:58 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=88
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/CiODBX5_NSk/
X-RSS-TAGS: Uncategorized

the v2.69 rss2email packages contain an example config.py file that might accidentally overwrite an existing config.py file. to keep this from happening i've updated both the [linux](http://www.allthingsrss.com/rss2email/rss2email-2.69.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.69.zip) by simply renaming the file to config.py.example.

[![](http://feedads.g.doubleclick.net/~a/v9hd_j59x3sfwg3q2xbjlncdusa/0/di)](http://feedads.g.doubleclick.net/~a/v9hd_j59x3sfwg3q2xbjlncdusa/0/da)  
[![](http://feedads.g.doubleclick.net/~a/v9hd_j59x3sfwg3q2xbjlncdusa/1/di)](http://feedads.g.doubleclick.net/~a/v9hd_j59x3sfwg3q2xbjlncdusa/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/ciodbx5_nsk)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/ciodbx5_nsk/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Minor correction to the minor correction: v2.69a released
Date: Mon, 13 Dec 2010 18:04:20 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=95
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/VgWvylH6fvk/
X-RSS-TAGS: Uncategorized

last week i made a [slight change to the v2.69 rss2email packages](http://www.allthingsrss.com/rss2email/2010/12/minor-correction-to-v2-69-packages/), which can cause suspicion that they have been tampered with. to mitigate this, i put the original v2.69 files back in place and renamed the updated package set to v2.69a.

here is v2.69a: [linux](http://www.allthingsrss.com/rss2email/rss2email-2.69a.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.69a.zip)  
and here is the original v2.69: [linux](http://www.allthingsrss.com/rss2email/rss2email-2.69.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.69.zip)

[![](http://feedads.g.doubleclick.net/~a/a817ydrxpe_g34wxu-sazq6uihm/0/di)](http://feedads.g.doubleclick.net/~a/a817ydrxpe_g34wxu-sazq6uihm/0/da)  
[![](http://feedads.g.doubleclick.net/~a/a817ydrxpe_g34wxu-sazq6uihm/1/di)](http://feedads.g.doubleclick.net/~a/a817ydrxpe_g34wxu-sazq6uihm/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/vgwvylh6fvk)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/vgwvylh6fvk/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Mac OS X and rss2email are Compatible
Date: Fri, 17 Dec 2010 20:54:27 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=99
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/cILvxh4ziQg/
X-RSS-TAGS: Uncategorized

rss2email user jon thompson has reported that linux rss2email works well under mac os x. this is good news, but not to surprising since apple based os x on unix.

technically the rss2email linux package usually works on other unix-derived oss, such as centos and bsd. they aren't officially supported, but we usually try resolve incompatibilities if possible.

[![](http://feedads.g.doubleclick.net/~a/n8i-3iyxnst7ki6avoy7j2dfjk8/0/di)](http://feedads.g.doubleclick.net/~a/n8i-3iyxnst7ki6avoy7j2dfjk8/0/da)  
[![](http://feedads.g.doubleclick.net/~a/n8i-3iyxnst7ki6avoy7j2dfjk8/1/di)](http://feedads.g.doubleclick.net/~a/n8i-3iyxnst7ki6avoy7j2dfjk8/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/cilvxh4ziqg)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/cilvxh4ziqg/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Version 2.70 Released
Date: Tue, 21 Dec 2010 20:00:49 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=106
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/0rUaRcCJ8dU/
X-RSS-TAGS: Uncategorized

version 2.70 of rss2email is now available for both [linux](http://www.allthingsrss.com/rss2email/rss2email-2.70.tar.gz) and [windows](http://www.allthingsrss.com/rss2email/rss2email-2.70.zip).

changes from the previous version:

  * improved handling of given feed email addresses to prevent mail servers rejecting poorly formed froms 
  * added x-rss-tags header that lists any tags provided by an entry, which will be helpful in filtering incoming messages 

complete list in the official [changelog](http://www.allthingsrss.com/rss2email/changelog).

[![](http://feedads.g.doubleclick.net/~a/3-mtadqlwiwgd3bgvmqymi2xayg/0/di)](http://feedads.g.doubleclick.net/~a/3-mtadqlwiwgd3bgvmqymi2xayg/0/da)  
[![](http://feedads.g.doubleclick.net/~a/3-mtadqlwiwgd3bgvmqymi2xayg/1/di)](http://feedads.g.doubleclick.net/~a/3-mtadqlwiwgd3bgvmqymi2xayg/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/0ruarccj8du)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/0ruarccj8du/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Version 2.71 Release plus Other Major Updates
Date: Fri, 04 Mar 2011 18:03:11 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=123
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/wsLSVDNzno0/
X-RSS-TAGS: Uncategorized

good news everyone! two important tools that rss2email depends on have recently received major upgrades: [feedparser](http://code.google.com/p/feedparser/) and [html2text](https://github.com/aaronsw/html2text). these should improve rss2email's ability to handle feeds with poorly formed html and other weirdness.

the rss2email application itself also needed to be upgraded some to support these. changes in this version:

  * upgraded to feedparser v5.01! (http://code.google.com/p/feedparser/) 
  * upgraded to html2text v3.01! (https://github.com/aaronsw/html2text) 
  * potentially safer method for writing feeds.dat on unix 
  * handle via links with no title attribute 
  * handle attributes more cleanly with override_email and default_email 

[![](http://feedads.g.doubleclick.net/~a/mpc_o2acgpykdqhi-imaop4lbs0/0/di)](http://feedads.g.doubleclick.net/~a/mpc_o2acgpykdqhi-imaop4lbs0/0/da)  
[![](http://feedads.g.doubleclick.net/~a/mpc_o2acgpykdqhi-imaop4lbs0/1/di)](http://feedads.g.doubleclick.net/~a/mpc_o2acgpykdqhi-imaop4lbs0/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/wslsvdnzno0)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/wslsvdnzno0/

SENT BY: "rss2email: Lindsey Smith" <user@rss2email.invalid>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
From: "rss2email: Lindsey Smith" <user@rss2email.invalid>
To: a@b.com
Subject: Mac OS, rss2email and scheduling with launchd
Date: Tue, 23 Aug 2011 15:57:37 -0000
Message-ID: <...@dev.null.invalid>
User-Agent: rss2email/...
X-RSS-Feed: allthingsrss/feed.atom
X-RSS-ID: http://www.allthingsrss.com/rss2email/?p=131
X-RSS-URL: http://feedproxy.google.com/~r/allthingsrss/hJBr/~3/0hb7cT3sLUM/
X-RSS-TAGS: Uncategorized

just a little fyi if you encounter this: macrumors user durlecs was having a [problem scheduling rss2email with launchd](http://forums.macrumors.com/showthread.php?t=1216694), mac os's built in replacement for cron and other unix services. it looks like he/she got it working in the end by adding the path to python into the path environment variable.

[![](http://feedads.g.doubleclick.net/~a/0q4e9-dkghhcjqmhuxc-_qjrgka/0/di)](http://feedads.g.doubleclick.net/~a/0q4e9-dkghhcjqmhuxc-_qjrgka/0/da)  
[![](http://feedads.g.doubleclick.net/~a/0q4e9-dkghhcjqmhuxc-_qjrgka/1/di)](http://feedads.g.doubleclick.net/~a/0q4e9-dkghhcjqmhuxc-_qjrgka/1/da)

![](http://feeds.feedburner.com/~r/allthingsrss/hjbr/~4/0hb7ct3slum)


url: http://feedproxy.google.com/~r/allthingsrss/hjbr/~3/0hb7ct3slum/
